types/types.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	sdk "github.com/algorand/go-algorand-sdk/types"
+	"github.com/algorand/go-algorand/ledger/ledgercore"
+)
+
+// A ValidatedBlock represents an Block that has been successfully validated
+// and can now be recorded in the ledger.
+type ValidatedBlock struct {
+	Block sdk.Block
+	// todo: replace when statedelta endpoint is available
```

Comment:
nit: other todo's in indexer are all capitals
```suggestion
	// TODO: replace when statedelta endpoint is available
```